docs/tables/aws_ec2_network_interface.md

```diff
@@ -64,3 +64,21 @@ from
 order by
   eni;
 ```
+
+### Get network details for each ENI
+
+```sql
+select
+  eni.network_interface_id,
+  vpc.vpc_id,
+  vpc.is_default,
+  vpc.cidr_block,
+  vpc.state,
+  vpc.account_id,
+  vpc.region
+from
+  aws_ec2_network_interface eni,
```

Comment:
```suggestion
  aws_ec2_network_interface e,
  aws_vpc v
```

@karanpopat I believe we normally use shorter aliases/names, as it's easy to see what each alias relates back to. @rajlearner17 is this aligned with our other queries?